Makefile

```diff
@@ -3,7 +3,7 @@
 
 all: install lint build
 
-POETRY_VERSION = 1.1.5
+POETRY_VERSION = 1.1.12
```

Comment:
Is this make file being used anywhere now? Can we just drop it entirely?

Reply:
The readme has make for the different functions, so lets keep it for now